Resources/Prototypes/_CD/Voice/lay_emotes.yml

```diff
@@ -19,6 +19,10 @@
     - laid down.
     - lies down
     - lies down.
+    - fall
+    - fall.
```

Comment:
I don't think "fall" should be here. "Urist McHands fall" is not really good english.